website/docs/jellyfish/api/poolpair.md

```diff
@@ -161,6 +161,29 @@ interface PoolShareOptions {
 }
 ```
 
+## poolSwap 
+Creates a poolSwap transaction.
```

Comment:
`Creates a pool swap transaction with given metadata` Like what you wrote in another file